client/my-sites/posts/main.jsx

```diff
@@ -50,3 +101,20 @@ module.exports = React.createClass( {
 	}
 
 } );
+
+export default connect( ( state ) => {
+	const selectedSite = getSelectedSite( state );
+	const siteId = selectedSite && selectedSite.ID;
+	const draftsQuery = {
+		type: 'post',
+		status: 'draft',
+		order_by: 'modified'
+	};
+
+	return {
+		drafts: getSitePostsForQueryIgnoringPage( state, siteId, draftsQuery ),
```

Comment:
I think we'd want full on infinite scroll here at some point. Probably when we refactor this into its own component and replace drafts drawer.